docs/products/opensearch/howto/upgrade-to-opensearch.rst

```diff
@@ -0,0 +1,45 @@
+Upgrade to OpenSearch
+=====================
+
+For current Aiven for Elasticsearch customers, upgrade is recommended as soon as possible so that we can continue to support your database requirements. Depending on your needs, the various options for testing and upgrading the service are detailed here. The upgrade does not require downtime, and any Kibana service alongside Elasticsearch will be upgraded to OpenSearch Dashboards at the same time.
```

Comment:
upgrade >> upgrading (after the comma in the first sentence)
OR:
upgrade is recommended >> we recommend that you upgrade

Reply:
Sounds much nicer, thanks